apps/src/templates/rubrics/AiAssessmentBox.jsx

```diff
@@ -73,50 +136,9 @@ export default function AiAssessmentBox({
             <StrongText>{i18n.aiAssessmentEvidence()}</StrongText>
           </BodyFourText>
           <ul>
-            {
```

Comment:
Pulled this out to a function to clean it up a bit and because adding the links was an extra complication.